primap2/pm2io/_conversion.py

```diff
@@ -18,11 +18,12 @@
 
 
 def convert_unit_primap_to_primap2(unit: str, entity: str) -> str:
```

Comment:
You didn't rename the function, though.

Reply:
There was another confirmation in the refactor dialogue that I missed.